beacon_node/lighthouse_network/src/discovery/enr.rs

```diff
@@ -165,63 +166,51 @@ pub fn create_enr_builder_from_config<T: EnrKey>(
         builder.udp6(udp6_port.get());
     }
 
-    if enable_libp2p {
```

Comment:
`Builder` is no longer exposed, and since this was only used here and in `boot` created two different versions